.github/workflows/build_and_test.yml

```diff
@@ -9,6 +9,10 @@ on:
     branches:
       - develop
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```

Reply:
Found this, https://github.com/nodejs/node/blob/818284b68737bbb9098cf12b1b221cbb24f06ce8/.github/workflows/build-tarball.yml#L27-L29

Not sure about just using .ref 🙈 I'm using the other one in here https://github.com/RocketChat/RC4Community/blob/d7c98d2eea15100b58adc3a209224526e59fb232/.github/workflows/build_and_test.yml#L8-L10 and so far working as expected.